internal/proxy/proxy_test.go

```diff
@@ -31,50 +37,81 @@ func TestApply(t *testing.T) {
 		ftp     string
 		socks   string
 		noProxy string
-		mode    string
+		auto    string
 
 		existingDirs  []string
 		existingPerms map[string]os.FileMode
 		prevContents  map[string]string
 		compareTrees  bool
 
+		glibMockNotRun        bool
```

Comment:
`glibMockNotRun` is rather a "want" value, no?

Reply:
You are using it in:
```go
			if !tc.glibMockError && !tc.glibMockNotRun && !tc.missingGlibExecutable {
				err := os.Remove(filepath.Join(root, glibCompileSchemasRunFile))
				require.NoError(t, err, "glib-compile-schemas mock was not executed")
			}
```

Which is the assertion part (or maybe you should check explicitely that the file is not create if the boolean is true), but this is a little bit like a `wantErr`, no?

Reply:
Maybe we should have one parameter `wantGlibMockRun` and then, keep the 2 others as fixtures input, wdyt?